.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+indent_style = tab
+end_of_line = crlf
```

Comment:
Instead of enforcing crlf which is a bit Windows-centric, consider a [.gitattributes](https://git-scm.com/docs/gitattributes) with 
```
* text=auto
```
which tells Git that anything it can detect as text, it will fix line endings according to platform and settings.

Reply:
Also, certain files like Tracker.lua are currently using spaces, consider running autoformatting on all files.